### ISSUE TITLE
v-model modifier on input ruins html

### ISSUE THREAD

**Issue by AuthorProxy** [oruga-ui/oruga#656]
Overview of the problem
Oruga version: [0.6.0]
Vuejs version: [3.2.47]
OS/Browser:

Description
trim modifier adds ugly `modelmodifiers="[object Object]"` to the final html:
`<o-input v-model.trim="test_model.property" name="test_name" />`

**Comment by mlmoravek** [comment]
@AuthorProxy I think v-model modifiers only work when used directly on an html input element. However, oruga components are just normal vue components. I don't know if v-model modifiers are also usable for components v-model. Do you have an example?

**Comment by blm768** [comment]
I don't think that the built-in `v-model` modifiers automatically work on anything but plain HTML elements, but it is possible to [get the set of modifiers used on a custom component](https://vuejs.org/guide/components/v-model.html#handling-v-model-modifiers).
